TESTS/mbed_hal_fpga_ci_test_shield/spi/main.cpp

```diff
@@ -57,10 +81,10 @@ void spi_test_common(PinName mosi, PinName miso, PinName sclk, PinName ssel, SPI
     tester.pin_map_set(ssel, MbedTester::LogicalPinSPISsel);
 
     // Initialize mbed SPI pins
-    spi_t spi;
+
     spi_init(&spi, mosi, miso, sclk, ssel);
     spi_format(&spi, sym_size, spi_mode, 0);
-    spi_frequency(&spi, 1000000);
+    spi_frequency(&spi, freqency);
```

Comment:
typo: `frequency`

Reply:
there are same typo in bellow code as well